rescript/dereplicate.py

```diff
@@ -13,16 +13,20 @@
 
 from q2_types.feature_data import DNAFASTAFormat
 
-from ._utilities import (run_command, _find_lca, _majority, _rank_handles,
+from ._utilities import (run_command, _find_lca, _majority,
                          _find_super_lca)
 
+from .ncbi import _allowed_ranks
+
 
 def dereplicate(sequences: DNAFASTAFormat,
                 taxa: pd.DataFrame,
                 mode: str = 'uniq',
                 perc_identity: float = 1.0,
                 threads: int = 1,
-                rank_handles: str = 'silva',
+                rank_handles: list = ['domain', 'phylum', 'class', 'order',
```

Comment:
I think I am actually using `_allowed_ranks` from `ncbi.py`. I should have just called values() on that instead of using `_allowed_rank_handles`. Though I did notice a mistake in [_allowed_ranks](https://github.com/mikerobeson/RESCRIPt/blob/bdd71bd0f2ebc8ba25055d59315b7f623112926e/rescript/ncbi.py#L39), both `family` and `forma` use `f__` as the rank handle.

Reply:
okay understood!